src/main/java/core/basesyntax/strategy/BalanceHandler.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public class BalanceHandler implements OperationHandler {
+    private static final String BALANCE = "b";
+
+    @Override
+    public List<FruitTransaction> getFruitFinalQuantity(List<FruitTransaction> listOfAll,
+                                                        List<FruitTransaction> listOfFruits) {
+        for (FruitTransaction listOfFruit : listOfFruits) {
```

Comment:
The variable name `listOfFruit` is misleading as it suggests a collection, but it's actually a single `FruitTransaction` object. Consider renaming it to `fruitTransaction` for clarity.